setup.py

```diff
@@ -108,10 +108,10 @@ def parse_readme():
             "Operating System :: MacOS",
             "Operating System :: Microsoft",
             "Operating System :: Unix",
-            "Programming Language :: Python :: 3.6",
             "Programming Language :: Python :: 3.7",
             "Programming Language :: Python :: 3.8",
             "Programming Language :: Python :: 3.9",
+            "Programming Language :: Python :: 3.10",
             "Topic :: Communications",
```

Comment:
Set minimum supported version to 3.6 and latest to 3.10

Reply:
minimum to 3.7 you mean?

Reply:
yep 😅